src/sass/partials/_global.scss

```diff
@@ -0,0 +1,55 @@
+body {
+  font-family: $fontPrimary;
+  color: $IIColor;
+  background-color: $IColor;
+  font-size: 14px;
+  letter-spacing: 0.03em;
```

Comment:
I think better is deleted letter-spacing

Reply:
I don't write this code just move it from main to partial scss, because this code here overrides changes in other files. This code belongs to you and was added in PR https://github.com/VasylDolishnii/goit-MainMasters/pull/4.